en/components/expansion_panel.md

```diff
@@ -34,19 +34,21 @@ We can now start using `igx-expansion-panel` in the markup. The expansion panel
         - `igx-expansion-panel-description` - stores the component description. Can be used to provide a short summary. Will always be displayed second in the headers.
```

Comment:
"always appear at the beginning of the header." Could be misleading as the icon location is not considered.